qutip/parallel.py

```diff
@@ -222,6 +222,8 @@ def parallel_map(task, values, task_args=tuple(), task_kwargs={}, **kwargs):
         each value in ``values``.
 
     """
+    if task_kwargs is None:
+        task_kwargs = dict()
```

Comment:
this can also be spelt `task_kwargs = task_kwargs or {}`

Reply:
This is auto-corrected by PyCharm, I'm fine with it unless shortening code is the way to go.

Reply:
It's less of a bother here, but in other functions you've changed there are 3 or 4 of them in a row which makes things more tedious to read.

Please try not to include automated changes in completely unrelated files when you're making commits (and PRs, to a lesser extent) - it's much easier in the review process and the commit history if these are separated.